sm/plugins/tedge_apt_plugin/tests/main.rs

```diff
@@ -0,0 +1,167 @@
+            transfer
+                .write_function(|data| {
+                    dst.extend_from_slice(data);
+                    Ok(data.len())
+                })
+                .unwrap();
+            transfer.perform().unwrap();
+        }
+        {
+            let mut file = File::create(&path)?;
+            file.write_all(dst.as_slice())?;
+        }
+        Ok(path)
+    }
+
+    #[test_case(
+        &format!("install {} --file {}", ROLLDICE, "wrong_path"),                                               // input
+        &format!("ERROR: Parsing Debian package failed"),                                                       // expected stderr
+        5                                                                                                       // expected exit code
+        ; "wrong path"                                                                                          // description
+    )]
+    #[test_case(
+        &format!("install {} --file {} --module-version {}", ROLLDICE, "notapackagepath", ROLLDICE_VERSION),    // input
+        &format!("ERROR: Parsing Debian package failed"),                                                       // expected stderr
+        5                                                                                                       // expected exit code
+        ; "wrong path with right version"                                                                       // description
+    )]
+    #[test_case(
+        &format!("install {} --file {} --module-version {}", ROLLDICE, ROLLDICE_FILE_PATH, "someversion"),      // input
+        &format!("ERROR: Parsing Debian package failed"),                                                       // expected stderr
+        5                                                                                                       // expcted exit code
+        ; "right path with wrong version"                                                                       // description
+    )]
+    #[test_case(
+        &format!("install {} --file {} --module-version {}", ROLLDICE, "notapackagepath", "someversion"),       // input
+        &format!("ERROR: Parsing Debian package failed"),                                                       // expected stderr
+        5                                                                                                       // expcted exit code
+        ; "wrong path with wrong version"                                                                       // description
+    )]
+    fn install_from_local_file_fail(
+        input_command: &str,
+        expected_stderr: &str,
+        expected_exit_code: i32,
+    ) -> Result<()> {
+        // no setup needed, wrong arguments are provided to tedge apt plugin
+        let (stdout, stderr, exit_code) = run_cmd(TEDGE_APT_COMMAND, input_command)?;
+
+        // asserting command failed
+        assert_that!(stdout, is(none()));
```

Comment:
It seems to be weird to convert an empty String to Option<String>, more idiomatic way to do this would be just call `.is_empty()` on the string.

```suggestion
        assert_that!(stdout.is_empty());
```